PassleSync.Core/UrlProviders/PassleAuthorUrlProvider.cs

```diff
@@ -13,8 +13,8 @@ public PassleAuthorUrlProvider(IRequestHandlerSection requestSettings, ILogger l
         {
         }
 
-        protected override string UrlPrefix => _configService.AuthorPermalinkPrefix;
-        protected override string ShortcodeName => "Shortcode";
+        protected override string UrlTemplate => _configService.PersonPermalinkTemplate;
+        protected override string ShortcodeName => "PersonShortcode";
```

Comment:
As above I'm concerned about Shortcode being renamed to PersonShortcode
Is this property being used to refer to the data coming from the Passle API?